types/stripe/index.d.ts

```diff
@@ -6977,20 +6977,22 @@ declare namespace Stripe {
     }
 
     namespace tokens {
-        interface IToken extends ICardToken, IBankAccountToken {}
+        type IToken = ICardToken | IBankAccountToken
 
         interface ICardToken extends ITokenBase {
+            type: "card";
```

Comment:
This should be single quotes.  Also there should be an empty line before the next comment.

Same goes for IBankAccountToken.